readme.md

```diff
@@ -5,14 +5,14 @@ manager for node. Although, npm does *not* stand for "node package manager".
 I won't go into the details of that, but if you'd like to, you can read all
 about it [here](https://docs.npmjs.com/misc/faq#if-npm-is-an-acronym-why-is-it-never-capitalized).
 
-npm makes it really easy to install, and maintain specific versions of
```

Comment:
Over emphasising the simplicity of a technical area the reader is studying can have unintentional negative impact. 
`npm` is far from easy.